images/scripts/virt-install-rhel4edge

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
```

Comment:
This file is copy of [virt-install-fedora](https://github.com/cockpit-project/bots/blob/main/images/scripts/virt-install-fedora) with edited kickstart file. 
I wonder how we want to proceed - `virt-install-fedora` is used for rhel-7, centos-7 and openshift, so never on fedora (anymore). I think we want to drop it eventually anyway. So should I cleanup this new file somewhat more and rename for example 'fedora.ks' to not mention fedora? Or should I just teach `virt-install-fedora` to also kickstart ostree images? (maybe put kicstart into separate file).

Reply:
Hmm, I was really hoping to get rid of all the kickstart and virt-install stuff enventually, but oh well.. Yes, let's make this a copy, and clean it up. virt-install-fedora should disappear in less than a year.